### PR TITLE
cura@4.8.0: Fix URL

### LINKED ISSUE #5231: cura@4.8.0: Download via aria2 failed
Download failed! (Error 3) Resource was not found

### LINKED ISSUE #5299: cura@4.8.0: download via aria2 failed
```
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'cura' (4.8.0) [64bit]
Starting download with aria2 ...
Download: Download Results:                                                                                                                               .7z
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 02cbcd|ERR |       0B/s|C:/Users/benja/scoop/cache/cura#4.8.0#https_github.com_Ultimaker_Cura_releases_download_4.8_Ultimaker_Cura-4.8.0-amd64.exe_dl.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/Ultimaker/Cura/releases/download/4.8/Ultimaker_Cura-4.8.0-amd64.exe#/dl.7z
    referer=https://github.com/Ultimaker/Cura/releases/download/4.8/Ultimaker_Cura-4.8.0-amd64.exe#/
    dir=C:\Users\benja\scoop\cache
    out=cura#4.8.0#https_github.com_Ultimaker_Cura_releases_download_4.8_Ultimaker_Cura-4.8.0-amd64.exe_dl.7z

ERROR & 'C:\Users\benja\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\benja\scoop\cache\cura.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=4068 --continue --summary-interval 0

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=cura%404.8.0%3a+download+via+aria2+failed
```

### PR DESCRIPTION
**Pull request by MarioGK**
- Closes #5231
- Closes #5299

### CODE REVIEW

**Comment by MarioGK** [comment]
Sorry for the 7bfd05a commit, i messed up my branches already reverted hopefully this will get merged soon

File: bucket/cura.json [hunk 1]
```diff
@@ -3,12 +3,9 @@
     "description": "Model editing tools for 3D printing",
     "homepage": "https://ultimaker.com/en/products/ultimaker-cura-software",
     "license": "LGPL-3.0-only",
-    "suggest": {
-        "Microsoft Visual C++ Redistributables 2015": "extras/vcredist2015"
-    },
```
Comment (Ash258) [inline bucket/cura.json:8]: Why did you remove this suggest? Where is stated that now the application do not require vcredist?

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by b-reich** [comment]
@MarioGK can you answer the question?

**Comment by MarioGK** [comment]
@b-reich Sorry, I was busy and forgot to answer, in the official web site's requirements and on their github, both don't mention any kind of dependency and I tried on a clean windows 10 vm just with scoop and cura installed and it didn't require anything else, that is why I removed

**Review by rasa — APPROVED** [review]

**Pull request merged**